src/tests/ftest/io/eight_servers.py

```diff
@@ -98,7 +98,7 @@ def test_sequential(self):
                           sequentially.
         Use Cases: Different combinations of 1/64/128 Clients and
                    1K/4K/32K/128K/512K/1M transfersize.
-        :avocado: tags=ior,eightservers,ior_sequential
+        :avocado: tags=all,daosio,large,ior_sequential
```

Comment:
This test is actually > large as it wants 8 servers and 8 clients, making it > 8 nodes which is what large is currently.